hackclub.com.yaml

```
@@ -3173,3 +3173,7 @@ high-seas:
   - ttl: 600
     type: CNAME
     value: cname.vercel-dns.com.
+Gharbia.hackclub.com:
```

Subdomains are not case sensitive
```suggestion
gharbia.hackclub.com:
```